app/service_invite/rest.py

```diff
@@ -121,3 +124,95 @@ def validate_service_invitation_token(token):
 
     invited_user = get_invited_user_by_id(invited_user_id)
     return jsonify(data=invited_user_schema.dump(invited_user)), 200
+
+
+@service_invite.route("/service/<service_id>/invite/request-for/<user_to_invite_id>", methods=["POST"])
+def request_user_invite(service_id, user_to_invite_id):
+    request_json = request.get_json()
+    user_requesting_invite = get_user_by_id(user_to_invite_id)
+    recipients_of_invite_request_ids = request_json["service_managers_ids"]
+    recipients_of_invite_request = [get_user_by_id(recipient_id) for recipient_id in recipients_of_invite_request_ids]
+    service = dao_fetch_service_by_id(service_id)
+    reason_for_request = request_json["reason"]
+    invite_link_host = request_json["invite_link_host"]
+    accept_invite_request_url = f"{invite_link_host}/services/{service.id}/users/invite/{user_requesting_invite.id}"
```

Comment:
I can't remember if we/Leo talked about this URL being sent in the request body rather than the API building it? 🤔